decidim-admin/spec/system/admin_manages_newsletters_spec.rb

```diff
@@ -113,6 +113,24 @@
       visit decidim_admin.preview_newsletter_path(newsletter)
       expect(page).to have_content("Hello Sarah Kerrigan! Relevant content.")
     end
+
+    it "sends a test email to the logged user" do
+      visit decidim_admin.newsletter_path(newsletter)
+      perform_enqueued_jobs do
+        click_link "Send me a test email"
+      end
+      expect(page).to have_content("Yay! Newsletter has been sent")
+      expect(last_email.subject).to include("A fancy newsletter for")
+    end
+
+    it "admin clicks on this index button and receives the test newsletter" do
```

Comment:
Related to my previous comment, I propose to add a context here too: 
```suggestion
    context "when admin clicks on the 'send me a test email' button in the index page" do
      it "sends a test email" do
```

Reply:
Done.

Reply:
The same here, it doesn't look like it was applied the suggestion.